docs/references/lightdash-cli.mdx

```diff
@@ -405,6 +407,34 @@ Run a single model and create or update its `schema.yml` file:
 lightdash dbt run --select mymodel
 ```
 
+### `lightdash download`
+
+Downloads all charts and dashboards from your Lightdash project as code. A `lightdash` directory is created in your dbt project and all of the charts and dashboards are written there as .yml files. 
+
+You can make changes to the code and upload these changes back to your Lightdash project. Content that's been downloaded as code can still be managed in the Lightdash UI. 
+
+**Examples:**
+
+Download all charts and dashboards from your project. 
+
+```bash
+lightdash download
+```
+
+### `lightdash upload`
+
+Uploads all charts and dashboards from the `lightdash/` directory in your dbt project to your Lightdash project. 
```

Comment:
in here , we should replace `all` with `only updated` 
Unless you specify --force